lib/html_parser.dart

```diff
@@ -654,26 +658,65 @@ class HtmlRichTextParser extends StatelessWidget {
                   buildContext,
                   onError: onImageError,
                 );
-                parseContext.rootWidgetList.add(Image.memory(base64.decode(
-                    node.attributes['src'].split("base64,")[1].trim())));
+                parseContext.rootWidgetList.add(InkWell(
+                  child: Image.memory(
+                    base64.decode(node.attributes['src'].split("base64,")[1].trim()),
+                    width: imageProperties?.width,
+                    height: imageProperties?.height,
+                    scale: imageProperties?.scale ?? 1.0,
+                    matchTextDirection: imageProperties?.matchTextDirection ?? false,
+                    centerSlice: imageProperties?.centerSlice,
+                    gaplessPlayback: imageProperties?.gaplessPlayback ?? false,
+                    filterQuality: imageProperties?.filterQuality ?? FilterQuality.low,
+                    alignment: imageProperties?.alignment ?? Alignment.center,
+                    colorBlendMode: imageProperties?.colorBlendMode,
+                    fit: imageProperties?.fit,
+                    color: imageProperties?.color,
+                    repeat: imageProperties?.repeat ?? ImageRepeat.noRepeat,
+                    semanticLabel: imageProperties?.semanticLabel,
+                    excludeFromSemantics: (imageProperties?.semanticLabel == null) ? true : false,
+                  ),
+                  onTap: onImageTap,
+                ));
               } else {
                 precacheImage(
                   NetworkImage(node.attributes['src']),
                   buildContext,
                   onError: onImageError,
                 );
-                parseContext.rootWidgetList
-                    .add(Image.network(node.attributes['src']));
+                parseContext.rootWidgetList.add(InkWell(
+                  child: Image.network(
+                    node.attributes['src'],
+                    width: imageProperties?.width,
```

Comment:
![image](https://user-images.githubusercontent.com/10363862/57968595-faa42100-796c-11e9-8a28-67f873442092.png)
I don't think there is a need for a null check.
The Image widget can take null width and height.

Reply:
Yeah, I know the width and height can be null, I'm just thinking more from the HTML side of things, for instance if someone has `<img src="..." width="100" height="50">` but doesn't specify a width or a height in the `imageProperties`, then the image should have the specified width and height rather than the default.

Reply:
Ok. I didn't think of this situation as I was usually using the CSS file for styling.